tools/inkscape_extension/visicut_export.py

```diff
@@ -384,3 +402,4 @@ def get_original_filename(filename):
     sys.exit(1)
 
 # TODO (complicated, probably WONTFIX): cleanup temporary directories -- this is really difficult because we need to make sure that visicut no longer needs the file, even for reloading!
+# - maybe add the PID od the running visicut, then we can detect orphaned temp direcories.
```

Comment:
```suggestion
# - Maybe add the PID od the running visicut, then we can detect orphaned temp direcories.
```

Silly text change to retrigger CI.

Reply:
You can just re-run failed actions directly, though.